### ISSUE TITLE
cv::dnn::readNetFromONNX  pnet load error

### ISSUE THREAD

**Issue by xinyi61** [opencv/opencv#15244]
- OpenCV => 4.1.1
- Operating System / Platform => centos x86_64
- Compiler => gcc
##### 
When I load the onnx model (converted from pytorch ) using cv::dnn::readNetFromONNX, it will get the "terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.1.1) /opt/notebook_files/usefull/opencv-4.1.1/modules/dnn/src/dnn.cpp:525: error: (-2:Unspecified error) Can't create layer "23" of type "Exp" in function 'getLayerInstance'"

the script i use to convert pytorch to onnx is:
example = torch.rand(1, 3, 12, 12, requires_grad=True).to(config.DEVICES)
torch_out = torch.onnx._export(pnet,
                                                     example, 
                                                     "pnet_model_final.onnx",
                                                      export_params=True)   

[pnet_Weights.pth](https://github.com/xinyi61/xinyi61.github.TEST/blob/master/pnet_Weights.pth)

[pnet_model_final.onnx](https://github.com/xinyi61/xinyi61.github.TEST/blob/master/pnet_model_final.onnx)

**Comment by adamhrv** [comment]
Similar problem `Can't create layer "649" of type "Exp" in function 'getLayerInstance'`. I'm trying to load the ONNX Yolo object detection model from https://github.com/ultralytics/yolov3
```
net = cv.dnn.readNetFromONNX('yolov3.onnx')
cv2.error: OpenCV(4.1.1) ../modules/dnn/src/dnn.cpp:525: error: (-2:Unspecified error) Can't create layer "649" of type "Exp" in function 'getLayerInstance'
```

Steps to repeat error:
```
import cv2 as cv
net = cv.dnn.readNetFromONNX('yolov3.onnx')
```
Download ONNX model from: https://drive.google.com/drive/folders/1uxgUBemJVw9wZsdpboYbzUN4bcRhsuAI
Using OpenCV 4.1.1 via `conda install -c conda-forge opencv`

**Comment by ThanosRoidis** [comment]
Any luck with that? I get the same error when loading an exponential layer from tensorflow and, as crazy as it sounds, I am starting to think that  `tf.exp()` is not supported yet or, at the very least, it can not be a final output layer...

**Comment by ansonku** [comment]
meet the same issue. any comments ?

**Comment by dkurt** [comment]
For `Exp` nodes with are part of SoftMax layer there is now a fusing logic: https://github.com/opencv/opencv/pull/16291.

Please tests these changes and add comments if you have more examples with `Exp` node usage. Thanks!

**Comment by hope-yao** [comment]
It seems that 4.3.0 still has this issue...

**Comment by dkurt** [comment]
@hope-yao, Feel free to open a new issue.
